userspace/libsinsp/plugin.cpp

```diff
@@ -50,6 +50,23 @@ using namespace std;
 
 static std::set<uint16_t> s_all_plugin_event_types = {PPME_PLUGINEVENT_E};
 
+static bool check_is_index(std::string& str)
+{
+	int length = str.length();
+	if(length == 0 || (length > 1 && str[0] == '0'))
```

Comment:
I think we can use a comment here, stating that indexes starting with `0` are considered strings.

Reply:
Thank you I added it